USHIFT-267: stubs for MicroShift product documentation (#50054)
openshift/openshift-docs: microshift_architecture/microshift-understanding.adoc

:_content-type: ASSEMBLY
[id="microshift-understanding"]
= Understanding MicroShift
include::_attributes/attributes-microshift.adoc[]
:context: microshift-understanding
toc::[]

> Learn about {product-title}, interacting with {product-title} the command-line interface (CLI) tools, consumption experience, and integration with {op-system-first}.

Reviewer: ```suggestion
Learn about {product-title}, interacting with the {product-title} command-line interface (CLI) tools, consumption experience, and integration with {op-system-first}.
```
Additionally, what is "consumption experience"? Is this something users are expected to know? It may be worthwhile to elaborate a little more here.

Author: There is not a separate command line tool for MicroShift, so I don't think the suggested change is quite what we want.

I know @kalexand-rh has made some changes to the text, but if "interacting with {product-title} the command-line" is from my original version I suspect it was supposed to say "via the command-line".

Reviewer 2: FYI to all, this content has changed in the docs I have worked on. These changes are not reflected here since I was waiting on this PR being conformed to the spreadsheet and merged so I can create new ones.

[id="microshift-understanding-about_{context}"]
== About MicroShift

> MicroShift is a distribution of {OCP} optimized for small form factor and edge computing.

Reviewer: ```suggestion
MicroShift is a distribution of {OCP} optimized for small form factors and edge computing.
```

> Edge devices deployed in the field pose very different operational, environmental, and business challenges from those of cloud computing. These challenges motivate different engineering trade-offs for Kubernetes at the far edge than for cloud or near-edge scenarios. MicroShift's design goals cater to this:

Reviewer: ```suggestion
Edge devices deployed in the field pose very different operational, environmental, and business challenges from those of cloud computing. These challenges motivate different engineering tradeoffs for Kubernetes at the far edge than for cloud or near-edge scenarios. MicroShift's design goals cater to these factors, including:
```

> * Make frugal use of system resources, such as CPU, memory, network, and storage

Reviewer: ```suggestion
* Minimizing the use of system resources, such as CPU, memory, network, and storage
```

> * Tolerate severe networking constraints

Reviewer: ```suggestion
* Tolerating severe networking constraints
```

> * Update securely, safely, quickly, and seamlessly, without disrupting workloads

Reviewer: ```suggestion
* Securely and seamlessly updating without disrupting workloads
```

> * Extend and integrate cleanly with edge-optimized operating systems, like Fedora IoT and {op-system-first}

Reviewer: ```suggestion
* Extending and integrating cleanly with edge-optimized operating systems, like Fedora IoT and {op-system-first}
```

> * Provide a consistent development and management experience with other distributions of {OCP}

Reviewer: ```suggestion
* Providing a consistent development and management experience with other distributions of {OCP}
```
